tools/json-api-client/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "declaration": true,
+    "declarationMap": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "noEmitOnError": true,
+    "allowSyntheticDefaultImports": true,
+    "moduleResolution": "node",
+    "target": "es2017",
+    "module": "es2015",
+    "lib": ["es2018", "dom"],
```

Comment:
Is this only going to be used client-side? If not, remove "dom"

Reply:
I would like to use it in CLI's if we could

Reply:
Yeah, lets remove this then so we dont make any DOM assumptions by accident